packages/synapse-react-client/src/style/components/query_filter/_query-wrapper-synapse-plot.scss

```diff
@@ -0,0 +1,14 @@
+.SynapsePlots {
+  padding-top: 5px;
+  > .SynapsePlot {
+    // Hack around Plotly.  If modebar is hidden, it still takes up space for some reason.
+    // Move plot up (closer to title), and make sure the title is on top.
+    .plot-container {
+      margin-top: -19px;
+    }
+    p {
+      position: relative;
+      z-index: 1000;
```

Comment:
I think this is a little too aggressive ;)

<img width="400" alt="image" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/90c626e4-1b7a-40e3-80ba-c99fcfe30062">

The z-index for the dropdown is also 1000 but I think you can pick anything in [1, 999]

Reply:
Done